create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -27,10 +27,14 @@ _doc:
 
 # Linters & tests
 
+format:
+	@docker-compose run --rm $(PROJECT_NAME)_app black $(PROJECT_NAME) --exclude $(PROJECT_NAME)/migrations -l 79
+
 mypy:
 	@docker-compose run --rm $(PROJECT_NAME)_app mypy $(PROJECT_NAME)
 
 lint:
+	@docker-compose run --rm $(PROJECT_NAME)_app black $(PROJECT_NAME) --exclude $(PROJECT_NAME)/migrations --check -l 79
```

Comment:
don't concat this commands

Reply:
remove from lint 'cause formatting is optional